nano/secure/blockstore_partial.hpp

```diff
@@ -756,6 +756,16 @@ class block_store_partial : public block_store
 		return make_iterator<nano::qualified_root, nano::block_hash> (transaction_a, tables::final_votes);
 	}
 
+	nano::store_iterator<nano::account, nano::account_info> accounts_last (nano::transaction const & transaction_a) const override
```

Comment:
I think these should follow the STL naming convention rbegin/rend http://www.cplusplus.com/reference/vector/vector/rbegin/

Since we prefix the table this would be accounts_rbegin(), accounts_rend().